src/pages/ReportSettingsPage.js

```diff
@@ -65,6 +66,19 @@ class ReportSettingsPage extends Component {
         ];
     }
 
+    /**
+     * @param {Object} report - the given report we're viewing settings for
+     * @param {Object|null} policy - the workspace the report is on, null if the user isn't a member of the workspace
+     * @returns {Boolean}
+     */
+    shouldDisablePublicRoomRename(report, policy) {
+        if (!policy || !ReportUtils.isPublicRoom(report)) {
+            return true;
+        }
```

Comment:
@thienlnam With the latest refactor you did, the function is not returning the same anymore. Now it returns `true` if the report is NOT a public room. Before it was returning `false`. Right?

Reply:
I understand that we returned `false` if `!ReportUtils.isPublicRoom(report)` because the validation checks below only made sense for public rooms.

Reply:
we want it to return false if the report isn't a public room. Cases for other types of reports will be handled in other conditions.

Reply:
I can fix it

Reply:
I can change it back to what it was before

Reply:
Also, it technically should be okay to skip the check for if the report is public since the policy will always be null for public rooms)